### PR TITLE
Refactor Routes codes

### PR DESCRIPTION
**Pull request by txiong4**
PLS review senpai

### CODE REVIEW

File: routes/logout.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+
+router.get("/", async(req, res) => {
+    res.clearCookie("user");
+    res.redirect("/login");
+  });
```
Comment (jimmoua) [inline routes/logout.js:7]: Run `npm run lint -- --fix`

File: routes/track.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const router = express.Router();
+const Package = require("../db/Models/Package");
+
+router.get("/", async(req, res) =>{
+    res.render("track");
+});
+router.post("/", async(req, res) =>{
+const userEmail = req.body.email;
+const findEmail = await Package.findAll({
+    where: {
```
Comment (jimmoua) [inline routes/track.js:11]: Spacing should be 2

Fix with lint

**Review by jimmoua — COMMENTED** [review]
Needs lint

File: routes/logout.js (hunk 1 above)
Comment (txiong4) [reply routes/logout.js]: fixed in 1ce2e7aca0bbd21a4f8091041c32c6a311e98dda

File: routes/track.js (hunk 2 above)
Comment (txiong4) [reply routes/track.js:11]: fixed in 1ce2e7aca0bbd21a4f8091041c32c6a311e98dda

**Review by jimmoua — APPROVED** [review]

**Pull request merged**